MessageViewController.podspec

```diff
@@ -8,5 +8,9 @@ Pod::Spec.new do |spec|
   spec.source       = { :git => 'https://github.com/GitHawkApp/MessageViewController.git', :tag => spec.version.to_s }
   spec.source_files = 'MessageViewController/*.swift'
   spec.platform     = :ios, '9.0'
-  spec.swift_version = '4.2'
+  if spec.respond_to?(:swift_versions) then
+    spec.swift_versions = ['4.2', '5.0']
```

Comment:
I considered putting 5.1 here, but this is supposed to match possible values for `SWIFT_VERSION`, and Xcode 11 doesn't show `5.1` as a possible value 🤷‍♂

Reply:
Weird! Do you have any theories on why?

Reply:
Best guess: it's not a major upgrade that would trigger Xcode to initiate migrations of any kind.

Reply:
Today I learned about supporting multiple Swift versions for one pod, neat!